package/src/handler.ts

```diff
@@ -713,13 +713,26 @@ export class Handler {
         }
         const searchToken = tokenResult.searchToken
 
+        const globalSymbolSearchSetting =
+            this.sourcegraph.configuration
+                .get()
+                .get('codeIntel.globalSymbolSearch') === undefined
+                ? true
+                : Boolean(
+                      this.sourcegraph.configuration
+                          .get()
+                          .get('codeIntel.globalSymbolSearch')
+                  )
```

Comment:
I find this ternary very complicated - you could just write:

```ts
this.sourcegraph.configuration.get().get('codeIntel.globalSymbolSearch') !== false
```

Reply:
Simplified ternary